### PR TITLE
fix: [CDS-48903]: Fixed artifactPath not getting resolved for docker artifact for artifactoryTags querry params

### PR DESCRIPTION
**Pull request by pankajsingh-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by dhruvUpadhyay07 — DISMISSED** [review]

**Review by samarth-sharma31 — APPROVED** [review]

File: src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/ArtifactSource/ArtifactoryArtifactSource/ArtifactoryArtifactSource.tsx [hunk 1]
```diff
@@ -459,9 +459,12 @@ const Content = (props: ArtifactoryRenderContent): JSX.Element => {
 
     return {
       artifactPath: getFinalQueryParamValue(
-        getImagePath(
```
Comment (samarth-sharma31) [inline src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/ArtifactSource/ArtifactoryArtifactSource/ArtifactoryArtifactSource.tsx:462]: Do we need to update the version  in package json for this hotfix ?

Comment (pankajsingh-harness) [reply src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/ArtifactSource/ArtifactoryArtifactSource/ArtifactoryArtifactSource.tsx:462]: @samarth-sharma31 there was some issue with this PR as some other commit has been pushed. Fixed it now.

**Comment by pankajsingh-harness** [comment]
retrigger jest

**Comment by YounixIO** [comment]
Original PR - https://github.com/harness/harness-core-ui/pull/13099

**Pull request closed without merging**